src/components/Sidebar/Sidebar.tsx

```
@@ -89,13 +89,14 @@ const SidebarProvider: FC<SidebarProviderProps> = ({
       }
 
       // This sets the cookie to keep the sidebar state.
+      // biome-ignore lint/suspicious/noDocumentCookie: it's necessary to set the cookie
```

Biome (1.9.4)

[error] failed to parse category "lint/suspicious/noDocumentCookie"



(suppressions/parse)